actions/parse-push/src/run.ts

```diff
@@ -0,0 +1,227 @@
+import * as core from '@actions/core'
```

Comment:
```suggestion
import { debug, getInput, warning } from '@actions/core'
```

I might have missed some, hard to tell in the web UI

Reply:
You did, but I do like this :)